frontend/src/pages/performers/PerformerDelete.tsx

```diff
@@ -50,42 +50,37 @@ const PerformerDelete: FC<Props> = ({ performer }) => {
     });
 
   return (
-    <>
-      <Form
-        className="PerformerDeleteForm"
-        onSubmit={handleSubmit(handleDelete)}
-      >
-        <Row>
-          <h4>
-            Delete performer <em>{performer.name}</em>
-          </h4>
-        </Row>
-        <Form.Control type="hidden" value={performer.id} {...register("id")} />
-        <Row className="my-4">
-          <Col md={6}>
-            <EditNote register={register} error={errors.note} />
-          </Col>
-        </Row>
-        <Row className="mt-2">
-          <Button
-            variant="danger"
-            className="ms-auto me-2"
-            onClick={() => history.goBack()}
-          >
-            Cancel
-          </Button>
-          <Button
-            type="submit"
-            disabled
-            className="d-none"
-            aria-hidden="true"
-          />
-          <Button type="submit" disabled={deleting}>
-            Submit Edit
-          </Button>
-        </Row>
-      </Form>
-    </>
+    <Form className="PerformerDeleteForm" onSubmit={handleSubmit(handleDelete)}>
+      <Row>
+        <h4>
+          Delete performer <em>{performer.name}</em>
+        </h4>
+      </Row>
+      <Form.Control type="hidden" value={performer.id} {...register("id")} />
+      <Row className="my-4">
+        <Col md={6}>
+          <EditNote register={register} error={errors.note} />
+          <div className="d-flex mt-2">
+            <Button
+              variant="danger"
+              className="ms-auto me-2"
+              onClick={() => history.goBack()}
+            >
+              Cancel
+            </Button>
+            <Button
+              type="submit"
+              disabled
+              className="d-none"
+              aria-hidden="true"
+            />
```

Comment:
This exists on all `*Delete` components. The actual submit button is below.
Can be removed? 🤔

Reply:
It's in all forms I think. The purpose was to prevent pressing enter from submitting the form. Since the disabled submit button is first it gets the keypress event, and silently does nothing. I can't reproduce the issue after removing them now though. At the time pressing enter in an event note would submit, but that doesn't seem to be the case, so maybe it was fixed in the form library? In either case, if you can't reproduce the issue then removing should be fine.

Reply:
Ah, I see.
Well, it doesn't happen on the delete forms solely because pressing enter on a `textarea`, being multiline, doesn't submit the form.
But it does still happen if you click enter on an input field on, say, performer form.